active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,184 @@
+ ~~~~ 
+     <local:Breadcrumb ItemsSource="{x:Bind Nodes}"> 
+        <local:Breadcrumb.ItemTemplate> 
+            <DataTemplate x:DataType="local:Node"> 
+                <local:BreadcrumbItem Content="{x:Bind Label}"/> 
+            </DataTemplate> 
+        </local:Breadcrumb.ItemTemplate> 
+    </local:Breadcrumb> 
+~~~~
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+# API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+## Notable Properties
+| Name | Description | Default |
+| :---------- | :------- | :------- |
+| ItemsSource | The items in Breadcrumb | Null |
+| ItemTemplate | to customize item visualization in the Breadcrumb | Null |
+
+## Events
+| Name | Description | 
+| :---------- | :------- | 
+| ItemClicked | Raised when a user interaction causes a jump to the CurrentItem.  |
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+## V1 API
+~~~~
+    public class Node 
+    { 
+        public string Label; 
+    } 
+App’s ViewModel: 
+
+    public IEnumerable<Node> Nodes { get; } 
+    
+public class Breadcrumb : Control 
+    { 
+        public Breadcrumb(); 
+
+        // Default: Null 
+        // Flat list of objects representing the nodes in current path. 
+        public Object ItemsSource { get; set; } 
+
+        // Default: Null 
+        // Allows to customize item visualization in the Breadcrumb. 
+        public Object ItemTemplate { get; set; } 
+
+        // Raised when a user interaction causes a jump to the CurrentItem. 
+        public event TypedEventHandler<Breadcrumb, BreadcrumbItemClickedEventArgs> ItemClicked; 
+
+        public static readonly DependencyProperty ItemsSourceProperty; 
+        public static readonly DependencyProperty ItemTemplateProperty; 
+    } 
+
+    public class BreadcrumbItem : ContentControl 
+    { 
+        public BreadcrumbItem(); 
+    } 
+
+    // Argument for the ItemClicked event. 
+    public sealed class BreadcrumbItemClickedEventArgs 
+    { 
+    // Returns the clicked item. It is either sender.CurrentItem  
+    // or an item in the Children list accessed for the dropdown of sender.CurrentItem. 
+        public object Item { get; } 
```

Comment:
The item might be ambiguous. For example if you're showing the path `\A\B\A`. We should include an index in here too.

Reply:
The item would be the data at that index. Providing the index sounds like a good idea to disambiguate.